c7n/resources/opensearch.py

```diff
@@ -181,17 +181,17 @@ def process_resource_set(self, client, resources, new_tags):
             client.tag_resource(Arn=r["PipelineArn"], Tags=tags)
 
 
-@OpensearchInjestion.action_registry.register('remove-tag')
-class RemoveTagOpensearchInjestion(RemoveTag):
-    """Remove tags from an OpenSearch Injestion Pipeline
+@OpensearchIngestion.action_registry.register('remove-tag')
+class RemoveTagg(RemoveTag):
```

Comment:
Guessing the intent here was:

```suggestion
class RemoveTagOpensearchIngestion(RemoveTag):
```